src/deno/Puppeteer.ts

```diff
@@ -139,6 +142,10 @@ export class PuppeteerDeno extends Puppeteer {
       } = {}
   ): Promise<Browser> {
     if (options.product) this._productName = options.product;
+    if (!options.executablePath && !this._downloaded) {
+      await installPuppeteer({ enableLog: false });
```

Comment:
I think this should still log out that it is going to install puppeteer. The script just hanging for 25 seconds on first use is kinda weird.

Reply:
@lucacasonato I added it so that it logs by default. I also added log levels. Minimal level will just print out a simple "Downloading chrome/firefox" message. Default logging (including progress bar) is still what it does right now. If you want to switch to using the "minimal" message, just change this: <https://github.com/Symbitic/deno-puppeteer/blob/main/src/deno/Puppeteer.ts#L149>
Otherwise, I can use `logLevel` as suitable for my use cases.